get rid of native dependencies (electron, electron-store, native-keymaps)

I've been trying to build the "next" version of several of the docker images in this repo. I was constantly getting build errors from `node-gyp` regarding the `native-keymap` module

```
> native-keymap@1.2.5 install /home/theia/node_modules/native-keymap
> node-gyp rebuild

Package x11 was not found in the pkg-config search path.
Perhaps you should add the directory containing `x11.pc'
to the PKG_CONFIG_PATH environment variable
Package 'x11', required by 'virtual:world', not found
Package 'xkbfile', required by 'virtual:world', not found
gyp: Call to 'pkg-config x11 xkbfile --libs' returned exit status 1 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at emitTwo (events.js:126:13)
gyp ERR! stack     at ChildProcess.emit (events.js:214:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:198:12)
gyp ERR! System Linux 4.9.125-linuxkit
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/theia/node_modules/native-keymap
gyp ERR! node -v v8.15.1
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! native-keymap@1.2.5 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the native-keymap@1.2.5 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-04-11T04_01_23_796Z-debug.log
```
**Solution**
The [solution](https://github.com/theia-ide/theia/blob/7d37b1245ab15f91ffdcf8606ffe7b5efe02ba2f/.gitpod.dockerfile#L11) already exists in the gitpod Dockerfile. Corresponding packages for `libx11-dev` and `libxkbfile-dev` must be added to each OS used under theia-apps.

Alpine: `apk add libx11-dev libxkbfile-dev`
Ubuntu: `apt-get install -y libx11-dev libxkbfile-dev`
CenttOS: `yum install -y libX11-devel libxkbfile-devel`

--- a/theia-docker/Dockerfile
+++ b/theia-docker/Dockerfile
@@ -13,7 +13,6 @@ RUN yarn --pure-lockfile && \
     echo *.ts.map >> .yarnclean && \
     echo *.spec.* >> .yarnclean && \
     yarn autoclean --force && \
-    rm -rf ./node_modules/electron && \
     yarn cache clean
 
 FROM node:${NODE_VERSION}-alpine

--- a/theia-go-docker/Dockerfile
+++ b/theia-go-docker/Dockerfile
@@ -12,7 +12,6 @@ RUN yarn --pure-lockfile && \
     echo *.ts.map >> .yarnclean && \
     echo *.spec.* >> .yarnclean && \
     yarn autoclean --force && \
-    rm -rf ./node_modules/electron && \
     yarn cache clean
 
 # See : https://github.com/theia-ide/theia-apps/issues/34

--- a/theia-swift-docker/Dockerfile
+++ b/theia-swift-docker/Dockerfile
@@ -43,7 +43,6 @@ RUN yarn --pure-lockfile && \
     echo *.ts.map >> .yarnclean && \
     echo *.spec.* >> .yarnclean && \
     yarn autoclean --force && \
-    rm -rf ./node_modules/electron && \
     yarn cache clean
 
 

--- a/yangster-docker/Dockerfile
+++ b/yangster-docker/Dockerfile
@@ -7,7 +7,6 @@ ADD $version.package.json ./package.json
 ARG GITHUB_TOKEN
 RUN yarn && \
     NODE_OPTIONS="--max_old_space_size=4096" yarn theia build && \
-    rm -rf ./node_modules/electron && \
     yarn cache clean
 
 FROM node:${NODE_VERSION}-alpine

